app/src/main/java/com/teampophory/pophory/feature/signup/SignUpButtonInterface.kt

```diff
@@ -0,0 +1,5 @@
+package com.teampophory.pophory.feature.signup
+
+interface SignUpButtonInterface {
+    fun setButtonState(state : Boolean)
+}
```

Comment:
```suggestion
fun interface SignUpButtonInterface {
    fun setButtonState(state : Boolean)
}
```

- (다음 PR에 반영) fun interface로 고쳐보고 어떤 점이 다른지 한번 봐
- (다음 PR에 반영) onChangeState로 함수명 바꿔도 좋을듯